build/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM quay.io/operator-framework/ansible-operator:v0.17.1
 USER root
 
+RUN pip3 install netaddr
```

Comment:
Do we need this package?

Reply:
Hugh, still we are not finalized with the policy so this PR is just a draft until we test & freeze it. 
For now this package is not required.